src/bin/cargo.rs

```diff
@@ -166,9 +169,54 @@ fn execute(flags: Flags, config: &Config) -> CliResult<Option<()>> {
         // For all other invocations, we're of the form `cargo foo args...`. We
         // use the exact environment arguments to preserve tokens like `--` for
         // example.
-        _ => env::args().collect(),
+        _ => {
+            let mut default_alias = HashMap::new();
+            default_alias.insert("b", "build".to_string());
+            default_alias.insert("t", "test".to_string());
+            default_alias.insert("r", "run".to_string());
+            let mut args: Vec<String> = env::args().collect();
+            if let Some(new_command) = default_alias.get(&args[1][..]){
+                args[1] = new_command.clone();
+            }
+            args
+        }
     };
 
+    match config.get_string(&format!("alias.{}", args[1])){
+        Ok(value) =>{
+            match value {
+                Some(record) => {
+                    // Group quoted items together
+                    let regex = Regex::new("(\"[^\"]*\"|[^\"]+)").unwrap();
```

Comment:
I can support both string and list. I like the string regex because its more user friendly and what someone might expect. There is no easy feedback for the final command that is ran. If someone needed to use the list version it might be hard to debug why the string is failing. Should I keep the regex and support the list or figure out an easy way to display the command that is ran?